README.md

```diff
@@ -79,6 +79,17 @@ cp *.auth /var/lib/uefistored/
 Production PCA 2011 from
 [here](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/windows-secure-boot-key-creation-and-management-guidance)
 
+## Logging
+
+`uefistored` supports defining log levels using the file /etc/uefistored/uefistored.conf.
+
+In /etc/uefistored/uefistored.conf, set the "loglevel" variable, such as:
+
+```
+# /etc/uefistored/uefistored.conf
+loglevel=DEBUG
+```
+
```

Comment:
Should we add the list of possible values in this doc?

Reply:
Yes.